sprangu-front/src/app/services/authentication.service.ts

```diff
@@ -8,14 +10,28 @@ export class AuthenticationService {
 
   user = new UserInfo();
 
-  constructor() {
+  constructor(
+    private router: Router
+  ) {
   }
 
   isLogged(): boolean {
     return this.user.id > 0;
   }
 
-  logout() {
+  logout(): void {
     this.user = new UserInfo();
+    localStorage.clear();
```

Comment:
Gal geriau išclear'int tik tuos laukus kuriuos buvai užset'inęs? Nes dabar ištrins viską, potencialiai ir iš kitų tab'ų svarbią info ar ne?